lib/cashramp/client.rb

```diff
@@ -12,7 +12,7 @@ class << self
   
       API_URLS = {
         live: 'https://api.cashramp.com/graphql',
```

Comment:
@benessilfie Please update this as well. The correct base URL is `api.useaccrue.com`